src/algebra/char_p/mixed_char_zero.lean

```diff
@@ -57,11 +57,16 @@ variables (R : Type*) [comm_ring R]
 
 /--
 A ring of characteristic zero is of "mixed characteristic `(0, p)`" if there exists an ideal
-such that the quotient `R ⧸ I` has caracteristic `p > 0`.
+such that the quotient `R ⧸ I` has caracteristic `p`.
+
+**Remark:** For `p = 0`, `mixed_char R 0` is a meaningless definition as `R ⧸ ⊥ ≅ R` has by definition
```

Comment:
Could you please rewrite the comment so that you have ≤ 100 chars on a line?

Reply:
I think that's an old commit and has already been fixed.